src/styles/style/card.scss

```
@@ -0,0 +1,101 @@
+.card {
+  width: $cardWidth;
+  min-height: $cardHeight;
+  border: 1px solid $cardColorBorder;
+  border-radius: $cardBorderRadius;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  padding: 0 16px;
+
+  &__top {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+    margin: 32px auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $cardColorProductCode;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  &__price-name {
+    color: $cardColorProductCode;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &__price-product {
+    color: $cardColorPriceProduct;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
```

Use scss nesting, fix it everywhere
```suggestion
  &__price {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    
    &-name {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }
```